documentation/API proposals/API_Proposal_IoT_SIM_Fraud_Prevention_Subscibition.md

```diff
@@ -0,0 +1,12 @@
+| **Field**                                 | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |     |
+| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| API family name                           | IoT SIM Fraud Prevention Subscibition                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |     |
```

Comment:
```suggestion
| API family name                           | IoT SIM Fraud Prevention Subscription                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |     |
```

Reply:
Same change to be applied in file name

Reply:
Hi @jgarciahospital, I have updated the spelling mistake, thank you for your review